core-bundle/tests/EventListener/Security/TwoFactorFrontendListenerTest.php

```diff
@@ -552,12 +547,7 @@ private function mockScopeMatcher(bool $hasFrontendUser, RequestEvent $event): S
     private function getRequestEvent(Request $request = null, Response $response = null): RequestEvent
     {
         $kernel = $this->createMock(Kernel::class);
-
-        if (null === $request) {
-            $request = new Request();
-        }
-
-        $event = new RequestEvent($kernel, $request, HttpKernelInterface::MAIN_REQUEST);
+        $event = new RequestEvent($kernel, $request ?? new Request(), HttpKernelInterface::MAIN_REQUEST);
```

Comment:
To apply same codestyle as (e.g.) in https://github.com/contao/contao/pull/3769/files#diff-5ced46fdf62feba7ee121f368149866f881e3047bdf98517b2cbad4ccc1e5487R277:

```suggestion
	$request ??= new Request();

        $event = new RequestEvent($kernel, $request, HttpKernelInterface::MAIN_REQUEST);
```